addons/project/views/project_sharing_templates.xml

```diff
@@ -9,7 +9,8 @@
     </template>
 
     <template id="project_sharing" name="Project Sharing View">
-        <iframe width="100%" height="100%" frameborder="0" t-attf-src="/my/project/{{ project_id }}/project_sharing"/>
+        <!--    We need to forward the request lang to ensure that the lang set on the portal match the lang delivered -->
+        <iframe width="100%" height="100%" frameborder="0" t-attf-src="/{{ request.context['lang'] }}/my/project/{{ str(project_id) }}/project_sharing"/>
```

Comment:
Wouldn't it be safer to check that 'lang' is in the context? Or is it always present?

Reply:
> Wouldn't it be safer to check that 'lang' is in the context? Or is it always present?

When there is no lang in the URL, it default to en_US.
You can try by just testing this with no additional language set :D

Reply:
Also if the key could be an empty string that would become `//my/project/{{ str(project_id) }}/project_sharing` which I am unsure it is a valid url.
But as you told me that could not happen, I am fine.